pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ApexJorjeLogging.java

```diff
@@ -7,11 +7,10 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import apex.jorje.parser.impl.BaseApexLexer;
-
 public final class ApexJorjeLogging {
     // note: that's a static/strong reference in order to avoid that the logger is garbage collected
-    private static final Logger APEX_LOGGER = Logger.getLogger(BaseApexLexer.class.getName());
+    // private static final Logger APEX_LOGGER = Logger.getLogger(BaseApexLexer.class.getName());
+    // TODO(b/239648780)
```

Comment:
The Summit AST parser also outputs extraneous info, such as
```
Aug 24, 2022 7:07:08 AM com.google.summit.translation.Translate translate
INFO: Translating <str>
Aug 24, 2022 7:07:09 AM com.google.summit.translation.Translate translate
INFO: Translated AST successfully. Created 1115 nodes.
Aug 24, 2022 7:07:10 AM com.google.summit.translation.Translate translate
INFO: Translating <str>
Aug 24, 2022 7:07:10 AM com.google.summit.translation.Translate translate
INFO: Translated AST successfully. Created 14 nodes.
```
, so it would also benefit from a different logging level (though the class should be renamed).

Reply:
ok, so I guess we still need something like that. Note - in PMD 7, we are using slf4j as the logging facade.